packages/swap/src/routes/__tests__/v2/swap.test.ts

```diff
@@ -1048,7 +1138,7 @@ describe('server', () => {
       expect(body.deposit.failure).toMatchObject({
         failedAt: 552000,
         failedBlockIndex: '92-1',
-        mode: 'DEPOSIT_TOO_SMALL',
+        mode: 'DEPOSIT_IGNORED',
```

Comment:
its a small breaking change, but i agree that its not very likely that somebody depends on this so no strong opinion

Reply:
could add it to the `CHANGELOG`

Reply:
apparently we don't check for it either 😅